tests/unit_tests/client/address_book/impl.cc

```diff
@@ -45,6 +45,46 @@
 /// @class SubscriptionFixture
 struct SubscriptionFixture {
 
+  struct AddressBookEntry
+  {
+   public:
+    AddressBookEntry() : m_Host(""), m_Address() {}
```

Comment:
Initializing a member string like this is unnecessary. Initializing `m_Address` like this is unnecessary because we do that during `Tag` construction.

Reply:
Wasn't sure here about initialization, so I added it just in case. Removed now.